handbook/rbac/rbac.md

```diff
@@ -0,0 +1,128 @@
+# Workspaces and Role-based Access Control
+
+In SystemLink 2020R2 NI introduced a new role-based access control (RBAC) system. This security technology provides strong isolation between different *workspaces* as well as fine grain privileges for systems, data, and analysis routines in SystemLink. Collectively this capability allows access to SystemLink to scale user and data access beyond a single team to an entire organization.
+
+## Concepts and Design
+
+SystemLink RBAC leverages common concepts such as *roles* and *users*. It also introduces concepts such as *workspaces* and *automatic data encapsulation* to ensure strict access control of data produced by test systems.
+
+<figure>
+    <img src="../../img/rbac-concepts.png" width="500" />
+    <figcaption>Role-based access control concepts in SystemLink</figcaption>
+</figure>
+
+### Workspace
+
+A workspace is a complete encapsulation of all systems, data, and other resources within SystemLink. If a user is not a member of a given workspace they cannot access or modify any of the resources within the workspace. This is true for access via the SystemLink web application and the SystemLink REST API. The SystemLink Advanced Server license is required to create multiple workspaces, but there is no limit on the number of workspaces that can be created. If a user is a member of multiple workspaces resource from each workspace will be shown together within the grids, dashboards, and other data views throughout SystemLink.
+
+<figure>
+    <img src="../../img/rbac-block-diagram.png" width="500" />
+    <figcaption>Two workspaces with unique resources in each workspace. The the resources users can access is determined by their workspace membership. The actions a user can take on each resource is determined by their role.</figcaption>
+</figure>
+
+Systems, data, analysis routines (collectively referred to as resources) are unique to individual workspaces. No resource occupies multiple workspaces. Some resources may be moved between workspaces. If a resource is duplicated the user is prompted to chose a workspace for the new resource.
+
+!!! note "Special Considerations for DataFinder"
```

Comment:
Is it worth mentioning other services that are not associated with workspaces?
While Files are associated with a workspace, File Moving rules are not. Notifications are not associated with a workspace.  Others?

Reply:
I added a section about the default workspace and another about exceptions to workspace rules to cover this.